StackOverflow/Controller/QuestionsTableViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  QuestionsTableViewController.m
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsTableViewController.h"
+#import "QuestionsCell.h"
+#import "DataFetchDelegate.h"
+#import "QuestionsDataObject.h"
+#import "QuestionsData.h"
+
+
+@interface QuestionsTableViewController () <DataFetchDelegate> {
+    NSArray *dataResult;
+    QuestionsDataObject *dataObject;
```

Comment:
Variable names should never end in Object, we know they are objects, in Objective C they can not be anything else. Once again remember your naming, variable must be named after their role, NOT their type. Data has a very specific meaning in iOS, it means an instance of NSData or in Swift a struct of type Data. If I see a name with data in it, I expect it to be one of these types not some arbitrary object. What is this ? Is it a Question, if it is a Question then call it that. If it is a list of Questions then call it listOfQuestions. This name does not actually telling me anything.